### DISCUSSION TITLE
Mapbox has not correct street names in turkey

### DISCUSSION THREAD

**Discussion by buraksecer** [rnmapbox/maps#3031] [Q&A]
We are using mapbox for Turkey users but some street names are not correct  or its  old. Is there a specific reason for this, for example, if we buy any plan and will it fix? 

For example: Seyrantepe, Gülizar st. No:31/A, 34418 Kâğıthane/İstanbul

but mapbox hasn't Gülizar st.

ps: mapbox is amazing! 🤩 

Best Regards.

**Comment by RyanTG** [comment]
Hi, I don't think this is the appropriate place to ask this question. This is a community-run mapbox package for using mapbox on react native. It has no control over mapbox content. 

My understanding is that a lot of the mapbox data comes from OpenStreetMap. If I were you I would check your areas on there to see if the source of the problem is OSM. If so, you can create an OSM account and directly correct the data.

If the problems do not stem from OSM data, then you might want to try contacting mapbox - but again, this repo will not put you in contact with mapbox.
